frontend/src/components/stylists/stylistSidebarRoutes.js

```diff
@@ -8,44 +8,78 @@ import {
   faSpa,
 } from "@fortawesome/free-solid-svg-icons";
 
-export function stylistSidebarRoutes(isActiveAppointment) {
-  return [
-    {
-      title: "Reservations",
-      path: "/stylists/reservations",
-      icon: faConciergeBell,
-      cName: "nav-text",
-    },
-    {
-      title: "Active Reservation",
-      path: "/stylists/activereservation",
-      icon: isActiveAppointment ? faExclamation : faSpa,
-      cName: "nav-text",
-    },
-    {
-      title: "Manage Schedules",
-      path: "/stylists/schedule/manage",
-      icon: faCalendarPlus,
-      cName: "nav-text",
-    },
-    {
-      title: "View Schedule",
-      path: "/stylists/schedule",
-      icon: faCalendarAlt,
-      cName: "nav-text",
-    },
+export function stylistSidebarRoutes(isActiveAppointment, stylist) {
```

Comment:
This change duplicates code. what is the purpose of it?

Reply:
Not sure were it duplicates code. I removed the active reservation view from the Manager since it wasnt working correctly giving a lot of 404s it was trying to get the current reservation from the manager which doesnt work as well. The only change was removing part of the view. Could have been done differently by perhaps removing it from the array but its a fix in the meantime if you guys can find a better way to work it